src/olympia/editors/views.py

```diff
@@ -686,7 +686,16 @@ def review(request, addon, channel=None):
     # The actions we should show a minimal form from.
     actions_minimal = [k for (k, a) in actions if not a.get('minimal')]
 
+    # Variables used to re-calculate auto-approval verdict.
+    auto_approval = {
+        'max_average_daily_users': int(
+            get_config('AUTO_APPROVAL_MAX_AVERAGE_DAILY_USERS') or 0),
+        'min_approved_updates': int(
+            get_config('AUTO_APPROVAL_MIN_APPROVED_UPDATES') or 0)
+    }
+
```

Comment:
You wouldn't need this elaborate dict if you called `calculate_verdict` before rendering the template too.